### ISSUE TITLE
redirect template points to sandbox version of afterpay.js

### ISSUE THREAD

**Issue by dwhoban** [newism/commerce-afterpay#7]
`src/templates/redirect.html.twig` contains a the line:

```js
<script src="https://portal.sandbox.afterpay.com/afterpay.js" async></script>
```

This does not work with production; and requires a variable to be created to honor the sandbox setting.

**Comment by leevigraham** [comment]
@dwhoban Did you have a PR / code snippet for this?

https://github.com/newism/commerce-afterpay/blob/master/src/responses/PurchaseResponse.php#L36

**Comment by dwhoban** [comment]
No not yet - haven't come back round to it

**Comment by leevigraham** [comment]
Fixed in b9d9ea2
